### ISSUE TITLE
App crashes when there are no available ECs

### ISSUE THREAD

**Issue by yeholerner** [kbbgl/e2ewd#51]
16/01/2020 09:33:11.072 [main] INFO  tests.MainTest - No ElastiCubes in RUNNING
mode.
Exception in thread "main" java.lang.IndexOutOfBoundsException: Index: 0, Size:0
        at java.util.ArrayList.rangeCheck(ArrayList.java:657)
        at java.util.ArrayList.get(ArrayList.java:433)
        at tests.MainTest.run(MainTest.java:140)
        at tests.MainTest.init(MainTest.java:52)
        at App.main(App.java:105)

**Comment by kbbgl** [comment]
We see that the token was invalid and caused a crash:
```
16/01/2020 09:33:10.602 [main] INFO  App - STARTING...
16/01/2020 09:33:10.797 [main] INFO  version.RepositoryComparator - e2ewd version: 9.1.0
16/01/2020 09:33:11.019 [main] INFO  tests.MainTest - Attempt number 1
16/01/2020 09:33:11.019 [main] INFO  tests.MainTest - 'runningRemotely=true'. Skipping rabbitmq health check...
16/01/2020 09:33:11.020 [main] INFO  tests.MainTest - Retrieving list of ElastiCubes...
16/01/2020 09:33:11.072 [main] WARN  t.q.ElastiCubeRESTAPIClient - Ensure that you have sufficient permissions to run calls to GET '/api/elasticubes/servers/LocalHost'
16/01/2020 09:33:11.072 [main] INFO  tests.MainTest - No ElastiCubes in RUNNING mode.
```

**Comment by kbbgl** [comment]
@yeholerner - What do you want to do if the token is invalid?

**Comment by kbbgl** [comment]
Added:
```
27/01/2020 21:38:52.170 [main] WARN  t.q.ElastiCubeRESTAPIClient - Ensure that you have sufficient permissions to run calls to GET '/api/elasticubes/servers/LocalHost'
27/01/2020 21:38:52.180 [main] INFO  tests.MainTest - Reason for failure: Permission denied with supplied token.
27/01/2020 21:38:52.182 [main] INFO  tests.MainTest - Test result: false
27/01/2020 21:38:52.183 [main] INFO  tests.MainTest - EXITING...
```

**Comment by kbbgl** [comment]
Fixed in 9.1.1.
